go/install/terminate.go

```diff
@@ -0,0 +1,21 @@
+// +build !ios,!android
```

Comment:
probably change the name of this file to reflect that it's platform specific

Reply:
to what?

Reply:
`terminate_desktop` or `terminate_nonmobile`? The main thing to avoid is confusion with the pattern where there's one common file that every platform builds, and then a bunch of platform-specific suffixed files to go with it.